src/settings-ui/Microsoft.PowerToys.Settings.UI/Views/PowerLauncherPage.xaml

```diff
@@ -226,12 +231,17 @@
                                            <Run x:Uid="PowerLauncher_AuthoredBy" />
                                            <Run FontWeight="SemiBold" Text="{x:Bind Author}" />
                                     </TextBlock>
+                                    <TextBlock 
+                                        x:Uid="PowerLauncher_NotAccessibleWarning"
+                                        FontSize="12"
+                                        Foreground="{ThemeResource SystemControlErrorTextForegroundBrush}"
+                                        Visibility="{x:Bind ShowWarning, Converter={StaticResource BoolToVisibilityConverter}}" />
```

Comment:
Should be 
`Visibility="{x:Bind ShowNotAccessibleWarning, Converter={StaticResource BoolToVisibilityConverter}}" />
`
in case we introduce other warnings later too.

Reply:
In such case, it would be easy to rename.